form_tests.feature

```diff
@@ -0,0 +1,169 @@
+Feature: Form Tests
+@Form_Tests @QA
+Scenario: Test incomplete form creation
+	#Using language.ccz instead of ccqa.ccz. Issues with accessing Basic Form of ccqa.ccz. Should only need to change a few lines/words in the first 2 scenarios of this test if I can bypass that issue. 
+	#Scenario covers Incomplete Forms 1 through 5
+	Then I install the ccz app at "languages.ccz"
+	Then I login with username "user_with_no_data" and password "123"
+	#Create Incomplete Form
+	Then I press start
+	Then I select module "Basic Form Tests"
+	Then I select module "Languages"
+	Then I wait
+	
+	Then I save form as incomplete
+	Then I wait to see "Start"
+	Then I open incomplete forms
+	Then I see the text "Languages"
+	
+	#Cancel Update of Incomplete form
+	Then I press "Languages"
+	Then I press "Go To Start"
+	Then I wait
+	#Press Enter a name to open keyboard
+	Then I press "Enter a name:"
+	Then I enter test
+	Then I go back to the home screen
+	
+	Then I open incomplete forms
+	Then I press "Languages"
+	Then I press "Go To Start"
+	Then I don't see the text "test"
+
+	Then I wait
+	Then I press "Enter a name:"
+	Then I enter test
+	Then I go back
+	Then I save form as incomplete
+	Then I wait
+	
+	Then I open incomplete forms
+	Then I press "Languages"
+	Then I press "Go To Start"
+	Then I see the text "test"
+	Then I exit form entry
+
+Scenario: Save in form from menu
+	#Incomplete Forms 6
+	Then I login with username "user_with_no_data" and password "123"
+	Then I press start
+
+	Then I select module "Basic Form Tests"
+	And I select form "Languages"
+
+	Then I enter test
+	Then I select "Save Form" menu item
+	Then I go back to the home screen
+
+	Then I wait
+	Then I open incomplete forms
+	Then I press "Languages"
+	Then I press "Go To Start"
+
+	Then I see the text "test"
+	Then I go back to the home screen
+
+Scenario: Form Entry Quirks
+	# Form Entry Quirks 1

```

Comment:
There was an issue with being able to save a form that violated a constraint (particularly, when a response was required). Added comments to clarify